docs/upgrading/upgrading-your-package.md

```diff
@@ -1,12 +1,12 @@
 # Upgrading Your Package
 
-Atom is rapidly approaching 1.0. Much of the effort leading up to the 1.0 has been cleaning up APIs in an attempt to future proof, and make a more pleasant experience developing packages.
+Atom is rapidly approaching 1.0. Much of the effort leading up to the 1.0 has been cleaning up APIs in an attempt to make developing packages future-proof and a more pleasant experience.
 
 This document will guide you through the large bits of upgrading your package to work with 1.0 APIs.
 
 ## TL;DR
 
-We've set deprecation messages and errors in strategic places to help make sure you don't miss anything. You should be able to get 95% of the way to an updated package just by fixing errors and deprecations. There are a couple of things you can do to get the full effect of all the errors and deprecations.
+We've set up deprecation messages and errors in strategic places to help make sure you don't miss anything. You should be able to get 95% of the way to an updated package just by fixing errors and deprecations. There are a couple of things you can do to gain visibility of all the errors and deprecations.
```

Comment:
I think "increase the visibility of all the errors and deprecations" would be even better.